--- ./deps/usrsctp/CMakeLists.txt
+++ ./deps/usrsctp/CMakeLists.txt
@@ -159,7 +159,7 @@
 #################################################
 
 # Determine if compiler is Visual Studio compiler or Clang in MSVC compatible mode
-if (CMAKE_C_COMPILER_ID MATCHES "MSVC" OR CMAKE_C_SIMULATE_ID MATCHES "MSVC")
+if (CMAKE_C_COMPILER_ID MATCHES "MSVC")
   set(C_COMPILER_IS_MSVC_LIKE TRUE)
 endif()
